docker/wgs_tertiary/scripts/json2ped.py

```diff
@@ -11,53 +11,53 @@
 6. phenotype (1=unaffected; 2=affected)
 """
 
-__version__ = "0.1.0"
+__version__ = "0.2.0"
 
 import json
 import csv
 import sys
 
 
 SEX = {"MALE": "1", "M": "1", "FEMALE": "2", "F": "2"}
-STATUS = {False: "1", True: "2"}
+STATUS = {False: "1", True: "2", "false": "1", "true": "2"}
 
 
 def parse_sample(family_id, sample):
-    """For a sample struct, return a list of PED fields."""
-    return [
-        family_id,
-        sample["sample_id"],
-        sample.get("father_id", "."),
-        sample.get("mother_id", "."),
-        SEX.get(sample.get("sex", ".").upper(), "."),  # all cases accepted
-        STATUS.get(sample.get("affected"), "0"),
-    ]
-
-
-def parse_family(family):
-    """For a family struct, return a list of lists of PED fields for each sample."""
-    family_id = family["family_id"]
-    samples = []
-    for sample in family["samples"]:
-        samples.append(parse_sample(family_id, sample))
-    return samples
+  """For a sample struct, return a list of PED fields."""
+  return [
+    family_id,
+    sample["sample_id"],
+    sample.get("father_id", "."),
+    sample.get("mother_id", "."),
+    SEX.get(sample.get("sex", ".").upper(), "."),  # all cases accepted
+    STATUS.get(sample.get("affected"), "0"),
```

Comment:
do you want to allow all cases for status as well? i.e. add `.upper()`, though this might not work if it’s properly coded to a boolean

Reply:
The workflow will fail immediately at runtime if caps are wrong, since `affected` is a `Boolean`.  This change is just meant to allow the script to accept `affected` if it has been coerced to `String`.